backend/src/api/private/notes/notes.controller.ts

```diff
@@ -261,9 +261,9 @@ export class NotesController {
     return await this.noteService.toNotePermissionsDto(returnedNote);
   }
 
-  @UseInterceptors(GetNoteInterceptor)
-  @Permissions(Permission.OWNER)
   @UseGuards(PermissionsGuard)
```

Comment:
I'm just noticing this is not needed here as we already have the annotation at the controller level…